docs/docs/deploying-to-kintohub.md

```diff
@@ -2,116 +2,43 @@
-
-2. Create a new **KintoBlock** of type **Website** and give KintoHub permission to access your GitHub repo.
-
-3. Select your GitHub repo from the list and press **Continue**.
-
-4. Choose **Static from Build** with the following values:
-
-   |                         |                                                            |
-   | ----------------------- | ---------------------------------------------------------- |
-   | **Name**                | `Gatsby Website` (or your own name)                        |
-   | **Language**            | `Node.js`                                                  |
-   | **Version**             | `latest` (or your own node.js version)                     |
-   | **Build Command**       | `npm install && npm run build` (or your own build command) |
-   | **Build Output Folder** | `public` (or your own output directory)                    |
-
-5. Press **Create Website**.
-
-The build will automatically start on the default GitHub branch of your repository.
-Wait for your build to be successful (green).
-
-#### Multiple Branches
-
-When you create a new **KintoBlock**, KintoHub imports all your branches and you can access them from your kintoblock page (new branches will also be automatically picked up).
-You can switch from one branch to another by pressing the branch name on the side panel on the left.
-
-#### Continuous Integration
-
-By default, KintoHub will build all the changes you push on your default branch.
-If you want to disable it:
-
-1. Disable the toggle **Build New Commits**.
-2. Press **Save Changes**.
-
-Your new changes will not be automatically picked up anymore and you will have to press **Build Latest Commit** to trigger a build.
-
-### Deploy your KintoBlock
-
-From your KintoBlock page, you will need to deploy your website first before accessing it.
-
-1. Press **Deploy Now** and then **New Project**.
-2. You can change the default values (optional).
-3. Press **Create New Project**.
-
-The deployment will start.
-Wait for it to be successful (green).
-
-_Note: you can disable Continuous Deployments for future builds._
-
-### Access your KintoBlock
+This guide assumes you already have a Gatsby project to deploy. If you need a project, you can use the [Quick Start](/docs/quick-start) or KintoHub's template [Gatsby Template](https://github.com/kintohub-examples/gatsby-site/generate) before continuing.
 
-From your Project page.
+## Deploy
 
-1. Scroll down to the **Added KintoBlocks** list.
-2. Press **Open** on your kintoblock (`Gatsby Website`).
+1. Navigate to [KintoHub](https://app.kintohub.com/) and login/sign-up your account.
 
-And that's it!
+2. Create a new service of type **Static Website**.
 
-## Deployment Configuration
+3. Grant the permission needed for accessing the repo that we created in the previous step. Or if you prefer not to connect your GitHub account, you may select **Import URL** and provide your access token.
```

Comment:
Per the [Style Guide](https://www.gatsbyjs.com/contributing/gatsby-style-guide/#use-you-as-the-pronoun), docs should use "you" instead of "we".

```suggestion
3. Grant the permission needed for accessing the repo that you created in the previous step. Or if you prefer not to connect your GitHub account, you may select **Import URL** and provide your access token.
```